docs/pages/bare/using-expo-client.md

```diff
@@ -1,49 +1,69 @@
 
-Inside a freshly initialized bare project, run `npx expo start`, and you can now run it in the client. Read this documentation to learn more about the limitations, why you might want to still use the client despite the limitations, and patterns you can apply to make this work well for you.
+The main drawback of developing in Expo Go is that it is impossible to send custom native code over-the-air to the Expo Go app, meaning you will need to do one of two things:
 
-## What are the limitations?
+1. Create a [development build](/development/introduction) to use custom native code, and distribute with [internal distribution](/build/internal-distribution) or TestFlight.
+2. Conditionally disable unsupported native features and distribute with Expo Go. This guide will demonstrate how achieve this.
 
-You will not be able to use the parts of your app that require custom native code. To run your bare app in Expo Go, you need to avoid calling any custom native code (native code that isn't included in the Expo SDK). For some apps this may mean that you won't be able to use the Expo Go app almost at all &mdash; for example, if your app depends on custom native code for something as fundamental as navigation or state management (eg: Realm or the Firebase native SDK) then not much of your app will be usable in the client. If your app only has some in app purchases, analytics, a custom map view, an AR view, and so on, then this may actually work great for you &mdash; that particular functionality would not be usable in Expo Go but the rest of the app still would be.
+## Usage
 
-## Why might you want to do this?
+Inside any React Native app, start a [development server with Expo CLI](/workflow/expo-cli#develop):
 
-There are a number of benefits to keeping your project runnable in the Expo Go app.
+<Terminal cmd={[
+"$ npx expo start",
+]} cmdCopy="npx expo start" />
 
-- Share your progress with stakeholders by publishing or sharing the development URL to see changes live
-- Continuously deploy builds from pull requests
-- No need to do native builds for iOS and Android in development because you use the Expo Go app instead
-- Develop the JavaScript side of your app from any machine of your choice, eg: use Windows for iOS development if you have an iOS device
-- Easily get new contributors set up on the project, only Node.js and a phone are required
-- You can use [Expo CLI](/workflow/expo-cli) for a great development experience
+<BoxLink title={<>Don't have <InlineCode>npx expo start</InlineCode>?</>} href="/bare/installing-expo-modules" description={<>Install and configure the <InlineCode>expo</InlineCode> package in your project.</>} />
 
-## Practical patterns for client-compatible bare apps
+Now you can launch your app in Expo Go by pressing `i` or `a` in the Terminal UI. Some features may cause your app to throw errors because certain native code is missing, continue reading to learn how you can conditionally skip unsupported APIs.
```

Comment:
```suggestion
Then, launch your app in Expo Go by pressing <kbd>i</kbd> or <kbd>a</kbd> in the Terminal UI. Some features may cause your app to throw errors because certain native code is missing, continue reading to learn how you can conditionally skip unsupported APIs.
```